cmd/clusterawsadm/cmd/bootstrap/iam/cloudformation.go

```diff
@@ -92,9 +92,14 @@ func createCloudFormationStackCmd() *cobra.Command {
 				return err
 			}
 
+			if err := resolveTemplateRegion(t, cmd); err != nil {
```

Comment:
If we remove the err check here, `t.Spec.Region` will be set to empty when there is no env var set, and passing an empty region to `session.Options` makes it resolve region from the default profile.

Same should be done for delete stack logic too.